test/cypress/integration/simplified-account/service-settings/worldpay-details/worldpay-details.cy.js

```diff
@@ -36,6 +37,7 @@ const setupStubs = (opts = {}, additionalStubs = []) => {
     gatewayAccountStubs.getAccountByServiceIdAndAccountType(SERVICE_EXTERNAL_ID, ACCOUNT_TYPE, {
       gateway_account_id: GATEWAY_ACCOUNT_ID,
       payment_provider: options.paymentProvider,
+      recurring_enabled: opts.recurringEnabled,
```

Comment:
I assume this should be `options.recurringEnabled`, otherwise the default value on line 24 isn't used. Not that it matters much since `opts.recurringEnabled` will be `undefined` if not provided, which is falsy

Reply:
admittedly I think I wrote that initially and could have chosen better names than `options` and `opts`